indi-mi/mi_ccd.cpp

```diff
@@ -992,3 +1002,50 @@ bool MICCD::saveConfigItems(FILE *fp)
 
     return true;
 }
+
+void MICCD::addFITSKeywords(fitsfile *fptr, INDI::CCDChip *targetChip)
```

Comment:
Please update the signature of this function to be inline with INDI Library [signature](https://github.com/indilib/indi/blob/328fefd1cfb32a5748a7da1bff441d2b0c3ae2c0/libs/indibase/indiccd.h#L508). targetChip can be accessed like how it is done in [CCD Simulator](https://github.com/indilib/indi/blob/328fefd1cfb32a5748a7da1bff441d2b0c3ae2c0/drivers/ccd/ccd_simulator.cpp#L1494)

Reply:
Ah, sorry, didn't have up-to-date indlib sources. It should be ok now.